### DISCUSSION TITLE
Welcome to libpsl Discussions!

### DISCUSSION THREAD

**Discussion by rockdaboot** [rockdaboot/libpsl#244] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ripperua** [comment]
Hi is libpsl compatible with android ?
I've prebuild staticaly and trying to link with curl and got erors 
```
[100%] Linking C executable curl
ld.lld: error: undefined symbol: __ctype_b_loc
>>> referenced by psl.c
>>>               libpsl_la-psl.o:(psl_str_to_utf8lower) in archive /home/ripper/sources_pck/android_libs/libpsl-0.21.5-android-x86_64-static/lib/libpsl.a
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [src/CMakeFiles/curl.dir/build.make:839: src/curl] Помилка 1
make[1]: *** [CMakeFiles/Makefile2:1150: src/CMakeFiles/curl.dir/all] Помилка 2
make: *** [Makefile:156: all] Помилка 2
```
